mav_msgs/include/mav_msgs/eigen_mav_msgs.h

```diff
@@ -252,6 +252,28 @@ struct EigenTrajectoryPoint {
   }
 };
 
+// Operator overload to transform Trajectory Points according to the Eigen
+// interfaces (uses operator* for this).
+// Has to be outside of class.
+// Example:
+// Eigen::Affine3d transform; EigenTrajectoryPoint point;
+// EigenTrajectoryPoint transformed = transform * point;
+inline EigenTrajectoryPoint operator*(const Eigen::Affine3d& lhs, const EigenTrajectoryPoint& rhs) {
+  EigenTrajectoryPoint transformed;
```

Comment:
Maybe it's better to copy the full point, i.e., `EigenTrajectoryPoint transformed = rhs;` and then transform all necessary points.

Reply:
Changed to copy constructor `EigenTrajectoryPoint transformed(rhs);`